deny.toml

```diff
@@ -103,6 +103,7 @@ allow = [
   "OpenSSL",
   "Unicode-DFS-2016",
   "Zlib",
+  "Unicode-3.0",
```

Comment:
@wilwade This appears to be compatible with our license but would be good to double check.